collection-spec/collection-spec.md

```diff
@@ -202,12 +202,17 @@ Clients only interested in the overall extent will only need to access the first
 It is recommended to only use multiple temporal extents if a union of them would then include a large
 uncovered time span (e.g. only having data for the years 2000, 2010 and 2020).
 
-Each inner array consists of exactly two dates and times, each in UTC.
-Each date and time MUST be formatted according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6).
+Each inner array consists of exactly two elements, either a timestamp or `null`.
```

Comment:
```suggestion
Each inner array consists of exactly two elements, each of which are either a datetime string formatted according to RFC 3339 or `null`.
```

Reply:
Discussing on the call to release - we only really use 'datetime' as our definition for the field in Item, not as a general way of talking about it. So I think we can just leave this as timestamp.